packages/frontend/src/components/Claim/WinBid/WinBidForm.tsx

```diff
@@ -38,22 +40,25 @@ export const WinBidForm = ({
   setVoucher,
 }: WinBidFormProps) => {
   const luck = win !== undefined
+  const { claimingEndTime } = useClaimingEndTime()
+  const FEE = 2
+  const withdrawalAmount = useMemo(() => bid.sub(bid.mul(FEE).div(100)), [bid])
```

Comment:
Should be `bid.mul(98).div(100)` to match contract logic. Results of `bid.mul(98).div(100)` and `bid.sub(bid.mul(FEE).div(100))` can differ when bid amount isn't divisible by 100.